dynamics-nav/endpoints-apis-for-dynamics.md

```diff
@@ -19,13 +19,14 @@ ms.author: solsen
 ## Accessing the endpoint
 Once you have the API access enabled, you can write code that integrates your web service or SaaS solution with [!INCLUDE[d365fin_long_md](includes/d365fin_long_md.md)]. Creating your integration through an API provides simple access to the supported functionality in a single endpoint, giving you a simplified experience for creating a single app with integrations across multiple Microsoft products.
 
-||[!INCLUDE[d365fin_long_md](includes/d365fin_long_md.md)]|||[!INCLUDE[d365fin_long_md](includes/d365fin_long_md.md)] On-Prem|
-|--|--|--|--|--|
+<!---||[!INCLUDE[d365fin_long_md](includes/d365fin_long_md.md)]|||[!INCLUDE[d365fin_long_md](includes/d365fin_long_md.md)] On-Prem|
```

Comment:
Why is this commented out? It looks like the links exist, but the syntax might be invalid.